test/Microsoft.AspNetCore.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -43,35 +42,6 @@ public class ContentResultTest
             MediaTypeAssert.Equal("text/plain; charset=utf-7", httpContext.Response.ContentType);
         }
 
```

Comment:
We need some more tests here, particularly around the boundary conditions of the buffer size/chunk size.